docs/reference/project/entrypoints.mdx

```diff
@@ -17,7 +17,8 @@ Entrypoints are defined in a projects nitric.yaml file.
 
 Supported resource types for entrypoints are:
 
-- [Services/Functions](./services)
+- [Containers](./containers)
```

Comment:
Are contains a new feature we going to support?  We currently dont have a containers.mdx file.

Reply:
Have put it on on the stack definition reference PR, they are currently in our nitric.yaml definition and we have build scripts/processes to support them :).